src/connectionController.ts

```diff
@@ -777,4 +782,69 @@ export default class ConnectionController {
   public setDisconnecting(disconnecting: boolean): void {
     this._disconnecting = disconnecting;
   }
+
+  private getСonnectionQuickPicks(): any[] {
+    if (!this._connections) {
+      return [
+        {
+          label: 'Add new connection',
+          data: {
+            type: NewConnectionType.NEW_CONNECTION
+          }
+        }
+      ];
+    }
+
+    return [
+      {
+        label: 'Add new connection',
+        data: {
+          type: NewConnectionType.NEW_CONNECTION
+        }
+      },
+      ...Object.values(this._connections)
+        .map((item) => ({
+          label: item.name,
+          data: {
+            type: NewConnectionType.SAVED_CONNECTION,
+            connectionId: item.id
+          }
+        }))
+        .sort((collectionA: any, collectionB: any) => {
+          if (collectionA.name < collectionB.name) {
+            return -1;
+          }
+          if (collectionA.name > collectionB.name) {
+            return 1;
```

Comment:
I think this sort might be doing not quite what we expect:
<img width="418" alt="Screen Shot 2020-07-28 at 10 24 31 AM" src="https://user-images.githubusercontent.com/1791149/88642892-5285fe00-d0c1-11ea-9348-08343c3daac0.png">

Looks like we're comparing `name` when the object builds the name into `label`, I think if we use `label` in the comparison it should be good.

Reply:
Interesting. I wanted it to be sorted the same way as in the sidebar. I assumed that it happens in `databaseTreeItem.ts` in the `getChildren` method that sorts by name. Is it some different list? Where we sort sidebar connections in code?

Reply:
Ah looks like collections aren't case insensitive also:
<img width="315" alt="Screen Shot 2020-07-28 at 11 28 11 AM" src="https://user-images.githubusercontent.com/1791149/88648336-308f7a00-d0c7-11ea-9db0-36f857455571.png">
I think most of the collections I have are lower case and never noticed it!
If you are up for fixing that as well that would be 🔥 we can also do it another time though.
Nice catch

Reply:
Np I will fix both places!

Reply:
Pushed updates, now connection quick pics and collections are case insensitive.